src/TypeSpec.Extension/Emitter.Csharp/test/Unit/modelType.test.ts

```diff
@@ -0,0 +1,114 @@
+import { TestHost } from "@typespec/compiler/testing";
+import assert, { deepStrictEqual } from "assert";
```

Comment:
Have you sort the imports?

Reply:
There is not really a feature to sort the imports. Since the prettier does not complain about this, I just let them be.
BTW I removed some unused import statements